entrypoint.sh

```diff
@@ -68,9 +68,13 @@ EXPORT_PATH=$TOFND_HOME/export
 echo "Using tofnd root:" $TOFND_HOME
 
 # add '--no-password' and '--unsafe' flags to args if enabled
-ARGS=${NOPASSWORD:+"${ARGS} --no-password"}
+ARGS=${NOPASSWORD:+"--no-password"}
```

Comment:
Why remove `ARG`? You're modifying the args env var so need to retain all of them. Or update the individual env var, and makes `ARGS` the concatenation?

Reply:
I add to remove `ARGs` because the arguments weren't getting concatenated at all. Suppose `ADDRESS` is set but `PORT` was not set. Everything concatenated up to `ADDRESS` would be there, but then `ARGS` would be replaced by an empty value. Instead, I am using the `+=` operator for assigning the returned value, which fixes the issue.

Reply:
conside removing lines 60-61 since `ARGS` is initialized here

Reply:
done!